File: src/contexts/codeimpact/tests/primaries.e2e_test/tests/fixtures/sample.cs
public class Sample
{
    public int Compute(int x)
    {
        if (x > 0)
        {
            for (int i = 0; i < x; i++)
            {
                DoWork(i);
            }
        }
        return x;
    }

    private void DoWork(int i)
    {
    }
}

File: src/contexts/codeimpact/tests/primaries.e2e_test/tests/fixtures/sample.ts
export class Sample {
  compute(x: number): number {
    if (x > 0) {
      for (let i = 0; i < x; i++) {
        fs.readFileSync(String(i));
      }
    }
    const label = x > 0 ? "positive" : "non-positive";
    return x;
  }
}
